inst/@string/string.m

```diff
@@ -588,6 +588,26 @@ function disp (this)
       out.tfMissing = cat (2, args_tfMissing{:});
     endfunction
 
+    ## -*- texinfo -*-
+    ## @node string.plus
+    ## @deftypefn {Method} {@var{out} =} plus (@var{a}, @var{b})
+    ##
+    ## String concatenation via plus operator.
+    ##
+    ## Concatenates the two input arrays, string-wise. Inputs that are
+    ## not string arrays are converted to string arrays.
+    ##
+    ## The semantics of concatenating missing strings with non-missing
```

Comment:
I think it would be nice to explicitly say that `plus` uses strcat, to make it really clear to the user that they have the same semantics, and the `+` operator is a convenience. Then you could just put a `@seealso{string.strcat}` at the bottom instead of duplicating the info from the strcat function here.